pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -972,4 +974,11 @@ public static class Range {
   public static class IdealState {
     public static final String HYBRID_TABLE_TIME_BOUNDARY = "HYBRID_TABLE_TIME_BOUNDARY";
   }
+
+  public static class RewriterConstants {
+    public static final String PARENT_AGGREGATION_NAME_PREFIX = "pinotparentaggregation";
+    public static final String CHILD_AGGREGATION_NAME_PREFIX = "pinotchildaggregation";
+    public static final String CHILD_AGGREGATION_SEPERATOR = "@";
+    public static final String CHILD_KEY_SEPERATOR = "_";
```

Comment:
Can you elaborate on purpose of this ?

Reply:
```
 /**
   * The name of the column as follows:
   * CHILD_AGGREGATION_NAME_PREFIX + actual function type + operands + CHILD_AGGREGATION_SEPERATOR
   * + actual function type + parent aggregation function id + CHILD_KEY_SEPERATOR + column key in parent function
   * e.g. if the child aggregation function is "argmax(0,a,b,x)", the name of the column is
   * "pinotchildaggregationargmax(a,b,x)@argmax0_x"
   */
```
To easily associate the child aggregation function with it's parents and extract the result with key.